docs/UserGuide.md

```diff
@@ -244,6 +248,14 @@ Format: `event -u DAYS`
 * The days refer to the number of days after the current date. All events that start within this time frame will be displayed. Events occurring on the day the command is entered will not be displayed.
 * The days **must be a positive integer less than 10000** 1, 2, 3, …​, 9999.
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about usage:**<br>
+
+DAYS entered is limited to 9999 days as this sufficiently covers the usage for most NUS students.
```

Comment:
Minor, but maybe a bullet point can be added here to keep it more in line with the other notes block